cou/steps/execute.py

```diff
@@ -89,25 +88,45 @@ async def _run_step(step: UpgradeStep, interactive: bool) -> None:
             await apply_plan(sub_step, interactive)
 
 
-async def apply_plan(plan: UpgradeStep, interactive: bool) -> None:
+async def apply_plan(plan: BaseStep, interactive: bool) -> None:
     """Apply the plan for upgrade.
 
     :param plan: Plan to be executed on steps.
-    :type plan: UpgradeStep
+    :type plan: BaseStep
     :param interactive:
     :type interactive: bool
     """
+    description = plan.description + " "
```

Comment:
Why we are adding this empty space `" "` here?

Reply:
For cosmetic purposes in generic cases. Otherwise if we have a single step that needs prompt, it will look like: `some step descriptionContinue(y/n)`

Reply:
I will prefer to have `" Continue(y/n)"` instead of adding `" "` here.  would that be possible? If not can you add some note here why it's there. Thanks

Reply:
The reason I didn't put the space before `Continue(y/n)` is to avoid unwanted indent in case of ApplicationUpgradePlan where `Continue(y/n)` is on a new line. I will go with your second suggestion and put a note here.